utils/json_profiles_merger_lib.py

```diff
@@ -41,7 +41,7 @@ def write_to_csv(
     output_csv_path: a path to the output CSV file.
   """
   output_dir = os.path.dirname(output_csv_path)
-  if not os.path.exists(output_dir):
+  if output_dir and not os.path.exists(output_dir):
```

Comment:
This is probably excessive. If `not output_dir` then `os.path.exists(output_dir)` would return False.

Reply:
This is required to check if the output_csv_path is a relative path to a file e.g. "output.csv"

```
output_csv_path = "output.csv"

output_dir = os.path.dirname(output_csv_path)
# output_dir = ""

not os.path.exists(output_dir)
# True

os.makedirs(output_dir)
# Stack trace...
# OSError: [Errno 2] No such file or directory: ''
```